### PR TITLE
Change failed to removed in cli

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Signed-off-by: mliyazud <mliyazud@redhat.com>

### CODE REVIEW

**Comment by MohamedAshiqrh** [comment]
@heketi/maintainers This is PR is result of discussion in https://github.com/heketi/heketi/pull/779

**Comment by MohamedAshiqrh** [comment]
This PR creates a temp variable for removed and replaces the failed to removed.

**Comment by MohamedAshiqrh** [comment]
@lpabon @obnoxxx PTAL

File: client/cli/go/cmds/node.go [hunk 1]
```diff
@@ -265,6 +265,10 @@ var nodeInfoCommand = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		const entryStateRemoved api.EntryState = "removed"
+		if info.State == api.EntryStateFailed {
+			info.State = entryStateRemoved
```
Comment (lpabon) [inline client/cli/go/cmds/node.go]: This is good, but quick question? Why create a const? Just in case it is used in the future?

Comment (MohamedAshiqrh) [inline client/cli/go/cmds/node.go]: Done 👍 :)

**Comment by MohamedAshiqrh** [comment]
@lpabon PTAL

**Comment by lpabon** [comment]
Sorry, that is not what I meant. I meant just use:

```go
info.State = "failed"
```

But it is ok, I'll just merge it and fix that later.

**Pull request merged**